auth-web/src/util/constants.ts

```diff
@@ -284,14 +282,14 @@ export enum LDFlags {
     IaSupportedEntities = 'ia-supported-entities',
     EnableFasDashboard = 'enable-fas-dashboard',
     DisableGovNAccountCreation='disable-govn-account',
-    EnableSpGpDba='enable-sp-gp-dba',
```

Comment:
I deleted this flag from this project. Once this code change is pushed to Prod, the flag can be deleted (archived) in LaunchDarkly.